docs/en/reference/sql/deployment_manage/ONLINE_SERVING_REQUIREMENTS.md

```diff
@@ -1,72 +1,76 @@
-# SQL On-Line Specifications and Requirements
+# Online Specifications and Requirements for SQL
 
-OpenMLDB Online Serving provides real-time feature extraction services. The [DEPLOY](../deployment_manage/DEPLOY_STATEMENT.md) command of OpenMLDB deploys a piece of SQL text to the wire. After the deployment is successful, users can perform feature extraction calculations on the request samples in real time through the Restful API or JDBC API. Not all SQL can be deployed to provide services online. OpenMLDB has a set of specifications for online statements and OP.
+OpenMLDB can provide real-time feature extraction services under *online request* mode. The [DEPLOY](../deployment_manage/DEPLOY_STATEMENT.md) command can deploy a piece of SQL text online. If the deployment is successful, users can perform feature extraction on the requested samples in real time through the Restful API or JDBC API. Note that only some SQL commands can be deployed to provide services online. To deploy these SQL commands please follow the specifications below.
```

Comment:
a piece of SQL text 
=>
a SQL script

Reply:
perform feature extraction on the requested samples in real time
perform real-time feature extraction